src/frontend/apps/bbb/DashboardMeeting/index.tsx

```diff
@@ -58,12 +58,36 @@ const DashboardMeeting = () => {
     { refetchInterval: meetingRefetchInterval.current },
   );
 
+  const consumerSiteUserId = `${getDecodedJwt().consumer_site}_${
+    getDecodedJwt().user?.id
+  }`;
+
+  useEffect(() => {
+    if (meeting?.infos?.attendees) {
+      for (const attendee of meeting.infos.attendees) {
```

Comment:
Instead of using `for const ... of ...` I suggest you to use [find](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/find)